tools/travis-setup-db.sh

```diff
@@ -39,7 +41,8 @@ if [ "$DB" = 'mysql' ]; then
         -v ${BASE}/${TOOLS}/docker-setup-mysql.sh:/docker-entrypoint-initdb.d/docker-setup-mysql.sh \
         -v ${SQL_TEMP_DIR}:${SQL_TEMP_DIR} \
         --health-cmd='mysqladmin ping --silent' \
-        -p 3306:3306 --name=mongooseim-mysql mysql
+        -p 3306:3306 --name=mongooseim-mysql \
+        mysql --default-authentication-plugin=mysql_native_password
```

Comment:
I think it'd be good to freeze the container version so that we don't have similar unexpected issue in the future when latest changes and doesn't run.